darts/timeseries.py

```diff
@@ -212,9 +212,9 @@ def copy(self, deep: bool = True) -> 'TimeSeries':
             A copy of this time series.
         """
         if deep:
-            return TimeSeries(self.pd_series(), self.conf_lo_pd_series(), self.conf_hi_pd_series())
+            return TimeSeries(self.pd_series(), self.conf_lo_pd_series(), self.conf_hi_pd_series(), self.freq())
```

Comment:
Maybe some warning in the constructor that the series is small and issues can occur would be useful

Reply:
done